src/resources/audit/AuditList.tsx

```diff
@@ -43,6 +43,21 @@ const SecurityRelatedFilter = ({
 
   return <Chip sx={{ marginBottom: 1 }} label={label} />
 }
+const availableResources = [
+  'user',
+  'item',
+  'batch',
+  'destruction',
+  'vaultLocation',
+  'dispatch',
+  'project',
```

Comment:
Aah, we cosmetically rename `project` using the config data.

I think this requires this change:
- array contains pairs of elements - the resource name and the cosmetically displayed label.  For this, we'll use the config value for `project` and present `vaultLocation` as `vault location` (same with mediaType).
- when we render values for the `resource` column, we should display this cosmetic label, not the raw `resource` value.

Reply:
Working on it.